docs/index.rst

```diff
@@ -16,7 +16,9 @@ Welcome to the Synapse documentation!
    synapse/quickstart
 
    synapse/userguide
-   synapse/devops
```

Comment:
since the devops index is removed, none of the old / existing devops docs are built in the docs.

Reply:
Yep, was planning to cut them out, but willing to discuss porting additional content in. Most of it was way to developer focused rather than truly devops focused.